presto-hive/src/main/java/com/facebook/presto/hive/HivePlugin.java

```diff
@@ -71,6 +73,12 @@ public void setNodeManager(NodeManager nodeManager)
         this.nodeManager = nodeManager;
     }
 
+    @Inject
```

Comment:
I think we should move away from injection for `Plugin` and make these explicit setters (now that default methods are allowed in interfaces).